CHANGELOG.md

```diff
@@ -44,6 +45,8 @@ v1.5.0-rc.0
 - (_Experimental_) Add a `prometheus.write.queue` component to add an alternative to `prometheus.remote_write`
   which allowing the writing of metrics  to a prometheus endpoint. (@mattdurham)
 
+- (_Experimental_) Add the function `arrary.combine_maps` to the stdlib. (@ptodev, @wildum)
+
```

Comment:
I'm adding it directly to the rc changelog to avoid another PR to main to switch it when we patch it